ojd_daps_skills/pipeline/extract_skills/extract_skills.py

```diff
@@ -259,14 +277,15 @@ def get_skills(self, job_adverts):
                 else:
                     skills[label].append(ent_text)
             predicted_skills.append(skills)
+
         return predicted_skills
 
-    def map_skills(self, predicted_skills):
+    def map_skills(self, predicted_skills:List[str]):
```

Comment:
Thinking about it, I think we might want to include the functionality here too. If I had a list of skills I'd expect to be able to use this function to map them. It feels like only a minor edition too

Reply:
ok gotcha - I'll add it to both then